larpandoracontent/LArHelpers/LArMCParticleHelper.cc

```diff
@@ -239,6 +239,42 @@ const MCParticle *LArMCParticleHelper::GetParentMCParticle(const MCParticle *con
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void LArMCParticleHelper::GetAllDescendentMCParticles(const pandora::MCParticle *const pMCParticle,
+        pandora::MCParticleList& descendentMCParticleList)
```

Comment:
Tend to prefer reference `&` to be next to variable, rather than the type. Check indentation.

Reply:
I'll probably do this a lot, this is a case of "old habits die hard". I always tended to treat this as type info attaching to the type, addressing and de-referencing attaching to the variable.